app/lib/actions/pulp3/content_view_version/create_exporter.rb

```diff
@@ -5,12 +5,14 @@ class CreateExporter < Pulp3::Abstract
         input_format do
           param :smart_proxy_id, Integer
           param :content_view_version_id, Integer
+          param :destination_server, String
         end
 
         def run
           cvv = ::Katello::ContentViewVersion.find(input[:content_view_version_id])
           output[:exporter_data] = ::Katello::Pulp3::ContentViewVersion.new(smart_proxy: smart_proxy,
-                                                                            content_view_version: cvv).create_exporter
+                                                                            content_view_version: cvv,
+                                                                            destination_server: input[:destination_server]).create_exporter
```

Comment:
destination_server is an attribute on content_view_version?

Reply:
ohh thats the content view version service class...  Thats confusing ;)

Reply:
question for you, would  'ContentViewVersionExport' be a better name for that?   or just "Export', or something clearer that that class is export related (yeah i know it came from your previous pr)

Reply:
Well I plan to use it for import also... Not sure CVVE is a better name

Reply:
alright, keep it as is for now.  If you get to import though, think about if it really belongs in that service class, or if a 2nd one makes more sense.

Reply:
Yeh totally agree..